app/src/main/java/com/example/flixster_metau/MainActivity.java

```diff
@@ -23,6 +23,7 @@
 import okhttp3.Headers;
 
 public class MainActivity extends AppCompatActivity {
```

Comment:
A `MainActivity` is usually a launching activity that jumps to/spawns other activities. Here since the Activity is primarily associated with a list of movies for user to browse, we typically name it semantically, e.g. something like `MovieListActivity`